agent/runtime-attach/README.md

```diff
@@ -0,0 +1,23 @@
+# Java runtime attachment for Application Insights
```

Comment:
or alternatively, we can expose this runtime attach feature on azure doc and we don't need a separate readme for it?

![image](https://user-images.githubusercontent.com/56097766/173453014-d7c4b221-836f-4b43-a12c-cf97eb4cccb1.png)

under `Advanced configurations`?

Reply:
@trask What do you think? May I create a PR on Azure repo to report the Readme documentation now or later?

Reply:
we can do that later when we're ready to make a new release.